### ISSUE TITLE
Error while trying to download song/album

### ISSUE THREAD

**Issue by miyukowo** [SwapnilSoni1999/spotify-dl#96]
Hello, I was trying to download some songs when suddenly, this error showed up
![image](https://user-images.githubusercontent.com/40559090/117833587-a785b000-b2a0-11eb-9908-f6d972e6aed3.png)

In order to counteract this, I tried creating another folder with the same name and restart the download. But it happened again, this time it's a metadata error. 
![image](https://user-images.githubusercontent.com/40559090/117834730-97220500-b2a1-11eb-9185-6b6563ec67fa.png)
![image](https://user-images.githubusercontent.com/40559090/117834875-b882f100-b2a1-11eb-96ee-a7aaea60bf19.png)

And apparently it only affects hololive songs and albums as I tried downloading an album and a single from another artist (TUYU) and it works just fine. Any reason why?
### Album - It's Raining After All
![image](https://user-images.githubusercontent.com/40559090/117835256-07c92180-b2a2-11eb-81fc-e55eb587557d.png)
### Single - Loser Girl
![image](https://user-images.githubusercontent.com/40559090/117835289-0f88c600-b2a2-11eb-8ce4-30cd5e84a5d0.png)

Finally, when I try to access the folder, it gives a location unavailable error, very weird. There's no way I can delete this folder I have to use a third-party program to force delete it.
![explorer_sNmjDfwl4u](https://user-images.githubusercontent.com/40559090/117835172-f849d880-b2a1-11eb-9f2d-1fc09fea01d7.png)

Also another thing is that the album name is Bouquet but the folder is named after the first song in the album (Dreaming Days), but this doesn't happen to other album from other artist. Weird.
### hololive - Bouquet
![image](https://user-images.githubusercontent.com/40559090/117835602-35ae6600-b2a2-11eb-9046-4995cd8a36fa.png)
### TUYU - It's Raining After All
![image](https://user-images.githubusercontent.com/40559090/117836737-0c420a00-b2a3-11eb-81d2-1b7646267907.png)

I hope to receive an answer to this problem.

**Comment by SwapnilSoni1999** [comment]
Can you please try from the latest version and update :) 
thanks

**Comment by phyzical** [comment]
hmm i wonder if this could be a windows issue? i.e the folder name contains a `.`

But as @SwapnilSoni1999 suggested if the issue still persists

could you please post the exact url to avoid copying incorrectly from the picture 👍 and we will try to replicate

**Comment by miyukowo** [comment]
Hey @SwapnilSoni1999 & @phyzical,
I updated to the latest version and tried, but unfortunately, the results are the same.
![image](https://user-images.githubusercontent.com/40559090/118360144-92669500-b5b0-11eb-8ad8-6baf3037acc9.png)
And while surfing through GitHub, I found out that someone was having the same problem when trying to download songs from this artist and it might have something to do with the Japanese letter or the UTF-8 unicode format thingy? That's what I've heard from them.
And here is the url you're asking for, [https://open.spotify.com/album/2m4GyMt1ffTR1XkA5gW4mR](https://open.spotify.com/album/2m4GyMt1ffTR1XkA5gW4mR)
I really appreciate your efforts. Thanks.
